_includes/_solo_boinc_intro.htm

```diff
@@ -25,13 +25,16 @@ <h4 class="text-bold">Install BOINC</h4>
         <!--Step 1-->
         <div class="row">
             <div class="col-6 col-sm-6 col-md-6 col-lg-6">
-                <p>Go to the <a href="https://boinc.berkeley.edu/download.php">BOINC website</a>, download and install BOINC for any computer you want to run BOINC on. Download the BOINC + Virtualbox option to ensure you can crunch the widest variety of projects possible. <b>Linux users: BOINC install packages are currently broken</b> please use <a href="https://www.reddit.com/r/BOINC4Science/comments/12h0g7v/quick_guide_installing_boinc_on_linux_right_now/" target="_blank">this workaround.</a></p>
+                <p>Go to the <a href="https://boinc.berkeley.edu/download.php">BOINC website</a>, download and install BOINC for any computer you want to run BOINC on. If you'll be crunching with multiple devices, please see <a href="/wiki/faq.html#can-i-crunch-boinc-on-multiple-machines-do-i-need-to-install-the-wallet-on-each-of-them">our FAQ</a>.
+<br><br>Windows users: Download the BOINC + Virtualbox option to ensure you can crunch the widest variety of projects possible. 
```

Comment:
what's with the double <br> tag?

Reply:
Gives it a little more spacing to separate this section out a bit.